analysis/urls/url.go

```diff
@@ -76,6 +78,27 @@ func getUrlCollectionScript(sysCfg *config.SystemConfig) (string, string, []stri
 	return source_collection_name, new_collection_name, keys, job, pipeline
 }
 
+// GetIPsFromHost uses the hostnames table to do a cached whois query
+func GetIPsFromHost(res *database.Resources, host string) []string {
```

Comment:
Often times we need to translate between domain name and ip address, here we provide a method to do so.